src/openrct2/paint/support/MetalSupports.cpp

```diff
@@ -156,27 +156,60 @@ static constexpr MetalSupportsImages _97B15C[] = {
     { 3334, 3353 }, // MetalSupportType::BoxedCoated
 };
 
-/** rct2: 0x0097B190 */
-static constexpr MetalSupportsImages _97B190[] = {
-    { 3243, 3226 }, // 3209
-    { 3279, 3262 },
-    { 3298, 3262 },
-    { 3334, 3317 },
-    {    0, 3658 },
-    {    0, 3658 },
-    {    0, 3141 },
-    {    0, 3158 },
-    {    0, 3175 },
-    {    0, 3192 },
-    {    0, 3124 },
-    { 3243, 3226 }, // 3209
-    { 3334, 3353 },
+/** 
+ * rct2: 0x0097B190
+ * Only the entries for “Tubes” and “TubesInverted” are different from the “A” map. 
+ */
+static constexpr ImageIndex kSupportBeamsB[] = {
+    3226, // MetalSupportType::Tubes (“A” map has 3209)
+    3262, // MetalSupportType::Fork
+    3262, // MetalSupportType::ForkAlt
+    3317, // MetalSupportType::Boxed
+    3658, // MetalSupportType::Stick
+    3658, // MetalSupportType::StickAlt
+    3141, // MetalSupportType::ThickCentred
+    3158, // MetalSupportType::Thick
+    3175, // MetalSupportType::ThickAlt
+    3192, // MetalSupportType::ThickAltCentred
+    3124, // MetalSupportType::Truss
+    3226, // MetalSupportType::TubesInverted (“A” map has 3209)
+    3353, // MetalSupportType::BoxedCoated
```

Comment:
Honestly think there should just be constants for these in some sprite ids file. Something for a future PR though.